org.freedesktop.Sdk.Extension.ocaml.yaml

```diff
@@ -0,0 +1,135 @@
+# flatpak-builder --force-clean build-dir org.freedesktop.Sdk.Extension.ocaml.yaml
+# flatpak-builder --user --install --force-clean build-dir org.freedesktop.Sdk.Extension.ocaml.yaml
+id: org.freedesktop.Sdk.Extension.ocaml
+branch: '23.08'
+runtime: org.freedesktop.Sdk
+build-extension: true
+sdk: org.freedesktop.Sdk
+runtime-version: '23.08'
+separate-locales: false
+appstream-compose: false
+build-options:
+  append-path: /usr/lib/sdk/ocaml/bin
+  prefix: ${FLATPAK_DEST}
```

Comment:
this is likely incorrect. 

1. ${FLATPAK_DEST} isn't yet defined
2. substitution isn't so you literally set it to `${FLATPAK_DEST}`

Reply:
As far as I can tell it is working properly:
- building the flatpak fails without the prefix
- all binaries are well located:
```
[📦 io.neovim.nvim ocaml]$ which ocaml
/usr/lib/sdk/ocaml/bin/ocaml
[📦 io.neovim.nvim ocaml]$ which dune
/usr/lib/sdk/ocaml/bin/dune
[📦 io.neovim.nvim ocaml]$ which opam
/usr/lib/sdk/ocaml/bin/opam
```

Do you think it would be better this way?
`  prefix: /usr/lib/sdk/ocaml`